srcpkgs/nyxt/template

```diff
@@ -12,22 +12,22 @@ short_desc="Keyboard-oriented, extensible web-browser"
 maintainer="0x0f0f0f <sudo-woodo3@protonmail.com>"
 license="BSD-3-Clause"
 homepage="https://nyxt.atlas.engineer/"
-distfiles="https://nyxt.atlas.engineer/static/release/nyxt-linux-${version}.tar.gz"
-checksum=71154bc6ef88f176f3f7783e1c0ecc191b6dcdd996e1c67f2bbc417b651e0355
+distfiles="https://github.com/atlas-engineer/nyxt/releases/download/${version}/nyxt-${version}-source-with-submodules.tar.xz"
+checksum=22cc8489e128b8695fbdf6796509c891bde630a73fd530c7582580adb3903773
 # Disable check because ASDF/USER::PROVE is not installed
 make_check=no
 nostrip=yes
 nopie=yes
 nocross="Cross-compilation fails due to package iolib/syscalls setting incorrect compiler flags"
 
 do_install() {
-	make PREFIX=/usr DESTDIR=${DESTDIR} install
-	vlicense licenses/ASSET-LICENSE
-	vlicense licenses/SOURCE-LICENSE
+    make PREFIX=/usr DESTDIR=${DESTDIR} install
+    vlicense licenses/ASSET-LICENSE
+    vlicense licenses/SOURCE-LICENSE
```

Comment:
We prefer tabs, I assume your editor did this automatically, could you fix it? When commiting the change, please use git commit --amend instead of git commit to have just one commit in this PR.

Reply:
Should be fixed now

Reply:
it isn't, did you `git add` the files before `git commit --amend`?

Reply:
@paper42 pretty sure i did, hmm, when you say tabs to you mean the actual tab character or will 4 spaces do? Because my editor ive been using inserts 4 spaces on tab

Reply:
We want literal tabs, you can use `xlint srcpkgs/nyxt/template` to check if the style is correct on your own computer.

Reply:
@paper42 ok now it's fixed